### PR TITLE
Add possibility to redefine setters for date components in the dxDateBox

### PR DESCRIPTION
**Pull request by mrjohn0011**

### CODE REVIEW

File: js/ui/date_box/ui.date_box.mask.js [hunk 1]
```diff
@@ -48,6 +48,13 @@ let DateBoxMask = DateBoxBase.inherit({
              */
             useMaskBehavior: false,
 
+            /**
+             * @name dxDateBoxOptions.dateComponentSetters
+             * @type object
+             * @default null
+             */
+            dateComponentSetters: null,
```
Comment (lukyanovas) [inline js/ui/date_box/ui.date_box.mask.js]: Take a look https://vuejs.org/v2/guide/computed.html#Computed-Setter

**Review by RomanTsukanov — DISMISSED** [review]

**Review by lukyanovas — DISMISSED** [review]
LGTM

File: js/ui/date_box/ui.date_box.mask.js [hunk 2]
```diff
@@ -381,6 +388,8 @@ let DateBoxMask = DateBoxBase.inherit({
     _optionChanged(args) {
         switch(args.name) {
             case "useMaskBehavior":
+            case "dateComponentGetters":
```
Comment (lukyanovas) [inline js/ui/date_box/ui.date_box.mask.js]: remove this dateComponentGetters

**Review by vconst — CHANGES_REQUESTED** [review]
Why?

**Pull request closed without merging**